source/gameshared/q_shared.c

```diff
@@ -705,6 +705,36 @@ const char *COM_RemoveColorTokensExt( const char *str, bool draw ) {
 	return cleanString;
 }
 
+/*
+* COM_UncolorBuffer
+*
+* Remove color tokens from the `in` buffer and write it to the `out`
+* buffer. `out` buffer must be at least as long as `in`.
+* It is valid to pass a buffer as both `in` and `out` to uncolor it
+* inplace.
+* Returns numer of chars written to the `out` buffer.
+*/
+int COM_UncolorBuffer( const char *in, int in_size, char *out ) {
```

Comment:
`Con_RemoveBufferColorTokens`?

Reply:
The return type as well as `in_size` parameters should be of `size_t` type (and that's what a caller uses).